--- /home/x2000habouzit/dev/banana/banana/spool.inc.php	2005-01-07 23:47:41.000000000 +0100
+++ banana/spool.inc.php	2005-01-07 23:46:58.000000000 +0100
@@ -150,7 +150,7 @@
         global $banana;
         $url = parse_url($banana->host);
         $file = $url['host'].'_'.$url['port'].'_'.$this->group;
-        return dirname(dirname(__FILE__)).'/spool/'.$file;
+        return "/var/spool/banana/$file";
     }
 
     function _updateSpool($arg)
